src/Symfony/Component/ErrorHandler/Exception/FlattenException.php

```diff
@@ -275,9 +275,10 @@ public function setTraceFromThrowable(\Throwable $throwable): self
     /**
      * @return $this
      */
-    public function setTrace($trace, $file, $line): self
+    public function setTrace($trace, $file = null, $line = null): self
     {
         $this->trace = [];
+        if ($file)
```

Comment:
I see that this is WIP :)

Reply:
I had to leave it after 2 hours that late last night. 

I would like to get some feedback from @yceruto and/or someone that knows about property access. 

Also, is it a good idea to modify a class "just because" serialization?